providers/dbt/cloud/src/airflow/providers/dbt/cloud/operators/dbt.py

```diff
@@ -239,7 +244,13 @@ def execute(self, context: Context):
                     DbtCloudJobRunStatus.CANCELLED.value,
                     DbtCloudJobRunStatus.ERROR.value,
                 ):
-                    raise DbtCloudJobRunException(f"Job run {self.run_id} has failed or has been cancelled.")
+
+                    raise DbtCloudJobRunDetailsException(
+                        self.account_id,
+                        self.run_id,
+                        message=f"Job run {self.run_id} has failed or has been cancelled.",
```

Comment:
How about 
```suggestion
                        message=f"Job run was {job_run_status}",
```

will this be more verbose for users ?

Reply:
That was the original message, and I think it's because this only gets raised if the status is `ERROR` or `CANCELLED`. However, I do think it would be cleaner to include `job_run_status` just in case there are additional statuses that get included in this block. Otherwise, we would have to update the message again to include any new statues.